src/store/assets/tokens.ts

```diff
@@ -73,11 +72,10 @@ export const nonFungibleTokensState = atom(get => {
 });
 export const stxTokenState = atom(get => {
   const balances = get(accountBalancesState);
-  if (!balances || balances.stx.balance.isEqualTo(0)) return;
   return {
     type: 'stx',
     contractAddress: '',
-    balance: new BigNumber(balances.stx.balance),
+    balance: balances?.stx.balance || 0,
```

Comment:
this is still meant to be a BigNumber, yeah? i though we spoke about this being the same as available balance?

Reply:
I guess this'll need to be changed when rebased on `feat/microblocks`

Reply:
this will likely be merged before that, so it will be the other way around

Reply:
`balances?.stx.balance` here is already `BigNumber`.

Reply:
I think Kyran's work just made this change in the `accountBalanceState`:
```
balance: new BigNumber(stxBalance || balances.stx.balance),
```

Reply:
> this is still meant to be a BigNumber, yeah? i though we spoke about this being the same as available balance?

That's what I did in my microblock branch after rebasing with main: both balance (anchored and unanchored) are BigNumber now.

Reply:
> I think Kyran's work just made this change in the `accountBalancesState`:
> 
> ```
> balance: new BigNumber(stxBalance || balances.stx.balance),
> ```

oh okay, perfect. What about it being the `currentAccountAvailableBalance`?

Reply:
When I switched to that it broke the integration test, not really sure why. I can do some more digging but I'm not sure what the difference is b/w the `accountBalancesState` balance and `currentAccountAvailableBalance`. The former seems to use an `accountInfoState` that I am unfamiliar with at this point.

Reply:
my understanding is that if we use just `accountBalanceState` it will not take into consideration the locked balance, which is what you changed in this PR, right?

Reply:
`stxBalance` in `accountBalanceState` does take into account the locked STX, so when I add that `or` condition to the `currentAccountAvailableStxBalanceState` all tests pass and everything is good to go. I will push it up.

Reply:
I have retested this everywhere and it seems to be behaving correctly in the UI and all tests are passing.